core/src/main/res/values/styles.xml

```diff
@@ -148,6 +157,13 @@
         <item name="ic_key">@drawable/ic_key_white</item>
         <item name="ic_volume_adaption">@drawable/ic_volume_adaption_white</item>
         <item name="scrollbar_thumb">@drawable/scrollbar_thumb_dark</item>
+
+        <item name="filter_dialog_button_background">
+            @drawable/filter_dialog_button_background_dark
+        </item>
+        <item name="filter_dialog_clear_active">@color/dialog_filter_clear_active_dark</item>
+        <item name="filter_dialog_clear_inactive">@color/dialog_filter_clear_inactive_dark</item>
+        <item name="filter_dialog_close">@color/filter_dialog_close</item>
```

Comment:
This should be a different resource for light and dark theme, I think

Reply:
<item name="filter_dialog_clear_active">@color/dialog_filter_clear_active_light</item>
        <item name="filter_dialog_clear_inactive">@color/dialog_filter_clear_inactive_light</item>

Reply:
They are. It gets confusing. color/filter_dialog_close.xml
```
<?xml version="1.0" encoding="utf-8"?>
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:color="?attr/filter_dialog_clear_active" android:state_checked="true" />
    <item android:color="?attr/filter_dialog_clear_inactive" />
</selector>
```
Or I can make two more files in the color directory like I've done with color radio button background.
filter_dialog_background_dark.xml
"" light.xml
Which is the only way you can do it with the radio button background attribute because for some reason the inflater doesn't want to inflate a selector with attributes on a background, possibly because of the custom of the custom radiobutton class?

Anyway, everything is in the color directory now.